src/current/_includes/releases/v24.3/v24.3.0.md

```diff
@@ -0,0 +1,109 @@
+## v24.3.0
+
+Release Date: November 18, 2024
+
+With the release of CockroachDB v24.3, we've added new capabilities to help you migrate, build, and operate more efficiently. Refer to our summary of the most significant user-facing changes under [Feature Highlights](#v24-3-0-feature-highlights).
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-0-feature-highlights">Feature highlights</h3>
+
+This section summarizes the most significant user-facing changes in v24.3.0 and other features recently made available to CockroachDB users across versions. For a complete list of features and changes in v24.3, including bug fixes and performance improvements, refer to the [release notes]({% link releases/index.md %}#testing-releases) for previous v24.3 testing releases. You can also search the docs for sections labeled [New in v24.3](https://www.cockroachlabs.com/docs/search?query=new+in+v24.3).
+
+- **Feature categories**
+    - [Change data capture](#v24-3-0-change-data-capture)
+    - [Migrations](#v24-3-0-migrations)
+    - [SQL](#v24-3-0-sql)
+    - [Operations](#v24-3-0-operations)
+    - [Observability](#v24-3-0-observability)
+
+- **Additional information**
+    - [Backward-incompatible changes](#v24-3-0-backward-incompatible-changes)
+    - [Key cluster setting changes](#v24-3-0-key-cluster-setting-changes)
+    - [Deprecations] (#v24-3-0-deprecations)
+    - [Known limitations](#v24-3-0-known-limitations)
+    - [Additional resources](#v24-3-0-additional-resources)
+
+{{ site.data.alerts.callout_info }}
+In CockroachDB Self-Hosted, all available features are free to use unless their description specifies that an Enterprise license is required. For more information, refer to the [Licensing FAQ](https://www.cockroachlabs.com/docs/stable/licensing-faqs).
```

Comment:
```suggestion
In self-hosted CockroachDB, all available features are free to use unless their description specifies that an Enterprise license is required. For more information, refer to the [Licensing FAQ](https://www.cockroachlabs.com/docs/stable/licensing-faqs).
```
I think we discussed making this sort of adjustment to terminology to align with the licensing changes? Ignore if I'm mistaken.